test/service/test_manifest.py

```diff
@@ -440,14 +440,37 @@ def test_manifest_zarr(self):
                 self.assertEqual(200, response.status_code)
                 lines = response.content.decode().splitlines()
                 file_prefix = 'output="587d74b4-1075-4bbf-b96a-4d1ede0481b2/'
+                location_prefix = f'url="{config.service_endpoint()}/repository/files'
                 curl_files = []
+                urls = []
+                related_urls = []
                 for line in lines:
                     if line.startswith(file_prefix):
                         self.assertTrue(line.endswith('"'))
                         file_name = line[len(file_prefix):-1]
                         curl_files.append(file_name)
+                    elif line.startswith(location_prefix):
+                        self.assertTrue(line.endswith('"'))
+                        url = furl(line[len(location_prefix):-1])
+                        try:
+                            url.args['drsPath']
+                        except KeyError:
+                            urls.append(url)
+                        else:
+                            related_urls.append(url)
+                    else:
+                        # The manifest contains a combination of line formats,
+                        # we only validate `output` and `url` prefixed lines.
+                        pass
                 self.assertEqual(sorted([f['file_name'] for f in expected]),
                                  sorted(curl_files))
+                self.assertEqual(len(expected) - 1, len(related_urls))
+                for url in related_urls:
+                    self.assertTrue({'drsPath',
```

Comment:
https://github.com/DataBiosphere/azul/blob/develop/CONTRIBUTING.rst#14line-wrapping-and-indentation

Reply:
@hannes-ucsc I extracted it into a variable instead. Using …
```python
self.assertTrue({
                    'drsPath', 
                    'fileName',
                    'requestIndex'
                }.issubset(url.args.keys()))
```
… seemed a bit off.